### ISSUE TITLE
Add meaningul description info

### ISSUE THREAD

**Issue by jimmybgammyknee** [ningbioinfostruggling/FeatureCount2GCT#2]
Can some meaningful annotation be added to the description field? Perhaps being able to download the gene information (HUGO gene name or function) from biomart using the gene names in the first field?

**Comment by ningbioinfo** [comment]
But the gct file format is pretty strict, it requires only 2 lines of description (the version line, and the row&col count line), because the following script used in the gtex pipeline is line-dependent(quite lazy). And the gene name in the outcome is depend on the the gene name used in feature count

**Comment by jimmybgammyknee** [comment]
Ah so its only for genome coordinates? I thought i saw an example which had functional information in that field...

**Comment by jimmybgammyknee** [comment]
Example on the site: https://software.broadinstitute.org/software/igv/sites/cancerinformatics.org.igv/files/linked_files/example.gct

**Comment by ningbioinfo** [comment]
Right, you mean the second column, I thought you mean the metadata like in the vcf files. I'll check biomart to see if I can make a dictionary obj to add that.

**Comment by jimmybgammyknee** [comment]
Maybe this? https://github.com/daler/biomartpy
